lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -70,6 +70,10 @@
         for the implicitly defined PUBLIC group.
       - 'Alias: I(role)'
     required: yes
+  target_roles:
+    description:
+      - Comma separated list of role (user/group) names to set as the default permissions.
```

Comment:
Based on official documentation from [postgresql.org],(https://www.postgresql.org/docs/9.4/sql-alterdefaultprivileges.html,)
I suggest to change the description:
```
Comma-separated list of existing role (user/group) names to set as the default permissions
for database objects subsequently created by them.
```
Or something similar

Reply:
Thanks for the hint, I changed that and add examples to the code, how to use target_roles